src/components/Error.svelte

```diff
@@ -1,15 +1,15 @@
 <script>
-import { error, reset } from '../data/error'
+import { error, reset } from 'data/error.js'
```

Comment:
just curious if .js is required in aliases. Also does vscode's autocomplete know how to use them?

Reply:
Yes, the file extension is required. I found a github issue that seemed to explain the situation. There was a debate about it, and the final decision was that it was a bad idea to not require the file extension. Seems it's a bit of a controversy and the side that likes the extensions obviously won the debate.

Just tested vscode and it seems to not recognize the aliases. Probably would require an update to the Svelte plugin or a separate plugin altogether.